### PR TITLE
chore(CI): Add check story book to CI and update storybook to v7

### PR DESCRIPTION
**Pull request by yanguoyu**
Because of https://github.com/storybookjs/storybook/issues/21642, [v6.5.17-alpha.0](https://github.com/storybookjs/storybook/releases/tag/v6.5.17-alpha.0) and v7 all fix this bug. So I update the storybook to v7.

And there are some deprecations from v6 to v7. https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#70-deprecations-and-default-changes. So I will open a new issue to resolve it.

### CODE REVIEW

**Review by WhiteMinds — APPROVED** [review]

File: packages/neuron-ui/.storybook/main.ts [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
+    name: "@storybook/react-webpack5",
+    options: {}
   },
   webpackFinal: config => {
     config.resolve.alias = {
       ...config.resolve.alias,
       electron: require.resolve('./electron')
-    }
-    return config
+    };
```
Comment (Keith-CY) [inline packages/neuron-ui/.storybook/main.ts:12]: Semicolons should be avoided.

Ref: https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-ui/.prettierrc.js#L6

**Comment by Keith-CY** [comment]
Irrelevant updates about formatting should be avoided

**Comment by yanguoyu** [comment]
> Irrelevant updates about formatting should be avoided

It will fail when running `npm run build-storybook`.

**Review by devchenyan — APPROVED** [review]

**Comment by Keith-CY** [comment]
> > Irrelevant updates about formatting should be avoided
> 
> It will fail when running `npm run build-storybook`.

Gotcha

**Review by Keith-CY — APPROVED** [review]

**Pull request merged**